app/lib/features/settings/pages/settings_page.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:acter/common/providers/common_providers.dart';
+import 'package:acter/common/themes/app_theme.dart';
+import 'package:acter/common/utils/routes.dart';
+import 'package:acter/features/settings/widgets/settings_menu.dart';
+import 'package:acter_avatar/acter_avatar.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:go_router/go_router.dart';
+
+class SettingsPage extends StatelessWidget {
+  const SettingsPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        _buildAppbar(),
+        Expanded(child: _buildBody()),
+      ],
+    );
+  }
+
+  AppBar _buildAppbar() {
+    return AppBar(
+      backgroundColor: Colors.transparent,
+      title: const Text('Settings'),
+    );
+  }
+
+  Widget _buildBody() {
+    return SingleChildScrollView(
+      child: Column(
+        children: [
+          _userProfileUI(),
+          const SettingsMenu(),
+        ],
+      ),
+    );
+  }
+
+  Widget _userProfileUI() {
+    return Consumer(
+      builder: (context, ref, child) {
```

Comment:
not necessary here either, if we made this a consumer-widget ... which I think, I'd prefer...